cdap-api/src/main/java/co/cask/cdap/api/dataset/lib/TimePartitionedFileSetArguments.java

```diff
@@ -54,6 +55,29 @@ public static Long getOutputPartitionTime(Map<String, String> arguments) {
   }
 
   /**
+   * This is the file extension for each partition when using TimePartitionedFileSet as an OutputFormatProvider.
+   * It's used to generate the end of the output file path for each partition.
+   * @param pathFormat The format for the path; for example: '%tF/%tH-%tM.%d',
```

Comment:
doc mentions that it is what `String.format()` uses.  One thing that should be mentioned in the javadoc is that the path must be unique per partition, otherwise runtime errors will occur.